### PR TITLE
Added intercom auth

### PR DESCRIPTION
**Pull request by srikar0896**

### CODE REVIEW

**Comment by pranav7** [comment]
@srikar0896 There are some failing specs on this one, if we can get that fixed, I'll try to spend some time today/tomorrow to role this out :)

**Comment by srikar0896** [comment]
@pranav7 yea, I am not sure why we are encountering `noMethodError`
image

Is this due to migration?

**Comment by pranav7** [comment]
https://github.com/pranav7/cadet-app/blob/618881b4ff2bf6017c87c6a1a496674d046388b7/config/initializers/devise.rb#L305

^ It's coming from this line. I think you might have the keys loaded in the test environment

File: config/cadet_config.example.yml [hunk 1]
```diff
@@ -7,6 +7,9 @@ development:
   slack: &slack
     channel: '#test'
     bot-api-token: ""
+  intercom: &intercom
+    client_id: ""
+    client_secret: ""
```
Comment (pranav7) [inline config/cadet_config.example.yml]: @srikar0896 Copying these to the test environment should fix your ci failure

**Pull request merged**